app/services/attachments/create_service.rb

```diff
@@ -32,14 +32,15 @@ def execute
 
     private
 
-    def identify_illumina_paired_end_files(attachments) # rubocop:disable Metrics/MethodLength,Metrics/CyclomaticComplexity,Metrics/AbcSize
+    def identify_illumina_paired_end_files(attachments) # rubocop:disable Metrics/MethodLength,Metrics/CyclomaticComplexity,Metrics/AbcSize, Metrics/PerceivedComplexity
```

Comment:
**nitpick (blocking):** Can you move the `_R1` `_R2` matching into a separate method called `identify_paired_end_files`. And just make them as `type: pe` instead of `illumina_pe` as they may or may not be illumina.

Reply:
Moved identifying regular pe files into their own method in 5e4fc6115a9cecf5130d295baaf4633dd3e1dcc4